lib/tasks/addon-install.js

```diff
@@ -37,7 +37,9 @@ class AddonInstallTask extends Task {
 
     let packageManager =
       commandOptions.packageManager ||
-      (commandOptions.yarn === true
+      (commandOptions.pnpm === true
```

Comment:
Maybe this should all move out to a utility (without the triple ternary), since it is in many places

Reply:
Can do!

Reply:
Done :tada: